path_planning/src/path_planning/ros_modules.py

```diff
@@ -87,6 +89,30 @@ def halt_and_catch_fire(self):
 
         self.controls_halted = True
 
+    def initialize_pneumatics_service(self, command):
+        if self.pneumatics_service is None:
+            rospy.wait_for_service('pneumatics_commands')
+            self.pneumatics_service = rospy.ServiceProxy('pneumatics_commands', PneumaticsCommands)
+        req = PneumaticsCommandsRequest()
+        req.command = command
+        response = self.pneumatics_service(req)
+        return response
+
+    def fire_l_torpedo(self):
```

Comment:
These functions should return the response so that the caller can use it if they want

Reply:
aad7325